sql/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java

```diff
@@ -523,25 +523,24 @@ public void testSysOperationsLogConcurrentAccess() throws Exception {
         Thread selectThread = new Thread(new Runnable() {
             @Override
             public void run() {
-                for (int i=0; i < 50; i++) {
-                    execute(
-                            "select count(*), race from characters group by race order by count(*) desc limit 2");
+                for (int i = 0; i < 50; i++) {
+                    execute("select count(*), race from characters group by race order by count(*) desc limit 2");
                 }
             }
         });
 
         Thread sysOperationThread = new Thread(new Runnable() {
             @Override
             public void run() {
-                for (int i=0; i < 50; i++) {
+                for (int i = 0; i < 50; i++) {
                     execute("select * from sys.operations_log order by ended");
                 }
             }
         });
         selectThread.start();
         sysOperationThread.start();
-        selectThread.join(500);
-        sysOperationThread.join(500);
+        selectThread.join(1000);
```

Comment:
would it make sense to make this dependend on the REQUEST_TIMEOUT value of `SQLTransportExecutor` ?